src/client/api/tests/job_tests.c

```diff
@@ -73,6 +73,7 @@ int uname(struct utsname *buf)
 		return -1;
 	}
 
+	memset(buf->nodename, 0, sizeof(buf->nodename));
```

Comment:
hmm, I think it would be better to just set the last character after the strcpy.   The only case where the result isn't null terminated is if the string is max length.  I guess it's just a test so the memset isn't on the critical path